include/freerdp/channels/camera-device.h

```diff
@@ -0,0 +1,338 @@
+/**
+ * FreeRDP: A Remote Desktop Protocol Implementation
+ * Video Capture Virtual Channel Extension
+ *
+ * Copyright 2022 Pascal Nowack <Pascal.Nowack@gmx.de>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FREERDP_CHANNEL_CAMERA_DEVICE_H
+#define FREERDP_CHANNEL_CAMERA_DEVICE_H
+
+#include <freerdp/api.h>
+#include <freerdp/dvc.h>
+#include <freerdp/types.h>
+
+#define CAM_DEVICE_ENUMERATOR_DVC_CHANNEL_NAME "RDCamera_Device_Enumerator"
+
+#define CAM_MSG_ID_SuccessResponse 0x01
+#define CAM_MSG_ID_ErrorResponse 0x02
+#define CAM_MSG_ID_SelectVersionRequest 0x03
+#define CAM_MSG_ID_SelectVersionResponse 0x04
+#define CAM_MSG_ID_DeviceAddedNotification 0x05
+#define CAM_MSG_ID_DeviceRemovedNotification 0x06
+#define CAM_MSG_ID_ActivateDeviceRequest 0x07
+#define CAM_MSG_ID_DeactivateDeviceRequest 0x08
+#define CAM_MSG_ID_StreamListRequest 0x09
+#define CAM_MSG_ID_StreamListResponse 0x0A
+#define CAM_MSG_ID_MediaTypeListRequest 0x0B
+#define CAM_MSG_ID_MediaTypeListResponse 0x0C
+#define CAM_MSG_ID_CurrentMediaTypeRequest 0x0D
+#define CAM_MSG_ID_CurrentMediaTypeResponse 0x0E
+#define CAM_MSG_ID_StartStreamsRequest 0x0F
+#define CAM_MSG_ID_StopStreamsRequest 0x10
+#define CAM_MSG_ID_SampleRequest 0x11
+#define CAM_MSG_ID_SampleResponse 0x12
+#define CAM_MSG_ID_SampleErrorResponse 0x13
+#define CAM_MSG_ID_PropertyListRequest 0x14
+#define CAM_MSG_ID_PropertyListResponse 0x15
+#define CAM_MSG_ID_PropertyValueRequest 0x16
+#define CAM_MSG_ID_PropertyValueResponse 0x17
+#define CAM_MSG_ID_SetPropertyValueRequest 0x18
+
+#define CAM_HEADER_SIZE 2
+
+struct _CAM_SHARED_MSG_HEADER
```

Comment:
We´ve fixed this struct definition style lately, better directly use `typedef struct` and not use reserved start symbols (`_`) in names

Reply:
Alright, changed.